v1/positions.go

```diff
@@ -71,3 +71,27 @@ func (b *PositionsService) Claim(positionId int, amount string) (Position, error
 
 	return *position, nil
 }
+
+// Closes a position with a market order
+func (b *PositionsService) Claim(positionId int) (Position, error) {
```

Comment:
Can we change this to `ClaimAll` or `ClaimFull` please? The 'Claim' function is already in use